### DISCUSSION TITLE
Problem Liquidity WanXRP/ WAN disappears

### DISCUSSION THREAD

**Discussion by Dams5** [wanswap/wanswap-interface#5] [Q&A]
Hi everybody,

I have a problem actually, i put some liquidity WanXRP/ WAN but i can't to see in Wanswap > Pool

See below the transaction: https://www.wanscan.org/tx/0x495e7f99b0cf35b343814a9f84b33863f5aaa192573c9ad09b233597c2bf11ca

Could you help me?

Thank you

**Comment by CaptainLogos** [comment]
I encountered the same problem! Just added liquidity to the same pool and am unable to see it. 😦😧


edit: I apologise for the false alarm, it is now visible under "Pool"
